.pre-commit-config.yaml

```diff
@@ -0,0 +1 @@
+repos: []
```

Comment:
let's add some repos here or remove this file?

Reply:
ah i see it's handled in https://github.com/2i2c-org/jupyterhub-home-nfs/pull/23/files, you can ignore this comment